File: eunji/0x03/BOJ_1475.cpp
#include <iostream>
using namespace std;


int main(void){
    int N;
    cin >> N;
    int counter[10] = {};
    // int target;
    while (N > 0){
        int temp = N % 10;
        counter[temp] ++;
        N /= 10;
    }
    int sixnine = counter[9] + counter[6];
    int sum;
    if (sixnine % 2 > 0) sum = sixnine / 2 + 1;
    else sum = sixnine / 2;

    int answer = 0;
    for (int i=0; i<10; i++){
        if ((counter[i] > answer) && (i != 9) && (i != 6)) answer = counter[i];
        // cout << counter[i];
    }
    // cout << sum << answer;
    cout << max(sum, answer);

}

File: eunji/0x03_array/BOJ_1475.py
N = int(input())

counter = {i:0 for i in range(10)}
while (N > 0):
    temp = N % 10
    counter[temp] += 1
    N //= 10
    
sixnine = counter[9] + counter[6]
sum = sixnine / 2 + 1 if (sixnine % 2 > 0) else sixnine /2

answer = 0
for i in range(10):
    if (counter[i] > answer) and (i!=6) and (i!=9):
        answer = counter[i]
        
print(int(max(sum, answer)))